### ISSUE TITLE
Some missing issues

### ISSUE THREAD

**Issue by basvroegop** [djfumberger/retromini#1]
Really love this theme and can't wait to see it evolve! I'll definitely be using it for a while.

When setting it up, I noticed some systems don't have proper assets yet. After going through my collection, I'm missing images for the following:

- Atari Lynx
- Atari 2600
- Dreamcast
- Famicom Disk System
- Game Boy Advance
- Game Gear
- Master System
- Nintendo DS
- PlayStation Portable
- Virtual Boy

This being version 0.1, I can imagine they're on your todo list already. But I figured I'd list them to be safe. Good luck!

**Comment by djfumberger** [comment]
Thanks for the list. Hopefully will get the missing ones created / added over the next few weeks. Current list based on requests is now:

* Atari Lynx
* Atari 2600
* Game Boy Advance
* Neogeo CD
* Pc Engine CD 
* Game Gear
* Master System
* PlayStation Portable
* Dreamcast
* Famicom Disk System
* Nintendo DS
* Virtual Boy

**Comment by djfumberger** [comment]
Added Game Boy Color b79861937128fc45d6dd91ad2c777a3126ced2bb

**Comment by djfumberger** [comment]
Added Game Boy Advance b9413a8f427e85347624cc4ee98244f37f1a779b

**Comment by djfumberger** [comment]
Added Game Gear d94fbfdbecaf02132414f26ba527c48e49ffe743

**Comment by djfumberger** [comment]
Added Dreamcast 7ce0a29a804c1da0dd8dc7b27c57e7d364d0c459

**Comment by djfumberger** [comment]
Added Master System b08d8657d6d8fcdc87dcc039a92b7cf547ab532e

**Comment by djfumberger** [comment]
Added NEOGEO CD 15ab5ed011105e85e6cf29b9129ba67e7abd8a05

**Comment by trvrplk** [comment]
A few more suggested additions:
- WonderSwan
- Computers (PC/DOS, Amiga, Atari ST)

**Comment by djfumberger** [comment]
Added Nintendo DS 3e5a88280f64c5f9ec19064e0d60ea4045afd251

**Comment by wariobar** [comment]
Thanks a lot! I would like to suggest a couple of arcade systems/groups: Atomiswave, Naomi, CPS1, CPS2, CPS3, FBA.

For some systems such as Megadrive 32x, Sega CD, or PC Engine CD I use the 'father system' image as the controller is the same. I think it looks great even repeating the image :)

**Comment by trvrplk** [comment]
You could pretty easily use the same image for all of those; I'd imagine it'd be a lot of work to make ones for all those individual arcade systems!

**Comment by MikhailovDenegir** [comment]
So I was able to add some images for missing systems by creating images and putting them in assets, images, system. However I’m not sure how to add the publisher to the system title. For example, PSP doesn’t say “SONY” above it. Not sure if that’s something under user control or if it needs to be added.

**Comment by bolognini** [comment]
I added some Mega Drive/Sega Genesis games and it renders as "Sega 8/16 bit (Various)" and doesn't have an image as well. Is it supposed to be that way because of some configuration I'm missing or is it the case of adding the console image?

And a silly question (first time with a device like Retroid): how those new added images comes to my system? I have to upgrade manually something or download Retromega again and substitute it on Retroid?

Btw, amazing work @djfumberger ! Such a beautiful layout for this frontend 🔥

**Comment by basvroegop** [comment]
@bolognini: That's probably because you're using the Pegasus config generator. If you change the collection and shortname in your metadata file, it should be all right.

I did notice Mega Drive doesn't get an image, because we technically only have a file for Genesis. Might be an idea to add a duplicate for Mega Drive so that always goes right.

**Comment by Clads92** [comment]
@djfumberger Hello, can you please add PSP Thank you !

**Comment by hh3k0** [comment]
> However, I’m not sure how to add the publisher to the system title.

I ran into the same issue, can't seem to find out where this is handled.

**Comment by cjwhitedev** [comment]
@djfumberger I submitted a PR with most of these missing images. 
For the rest of you, my fork can be found [here](https://github.com/cjwhitedev/retromega) in the mean time.

**Comment by chrisasma103** [comment]
Now that the Retroid pocket 2 plus is coming out, maybe we should add some more systems to account for the new consoles the RP2+ can play (even if it can only play them poorly.)

- Sega Saturn
- Nintendo Gamecube
- Nintendo 3DS
Let me know if I'm forgetting any more consoles the RP2+ can play.

**Comment by jeremyhalin** [comment]
Well, new RP2+ will have its own frontend and launcher, so pegasus won't be necessary anymore.

**Comment by hh3k0** [comment]
@jeremyhalin Well, will the front end and launcher be open source? I doubt it, considering they even refused to send me their makefile to compile some RetroArch cores. And the RP2 OS was bloated garbage, so I expect RP2+ OS to be as well. Rooted LineageOS with K-Tweaks is the way to go. Long story short: Pegasus will still be necessary.

**Comment by jeremyhalin** [comment]
Didn't know they were doing this in the shadows, that's sad. I guess you're right. Don't see the point buying it for me then. I'd rather go with a recalbox compatible handled machine. RP2 was ok but still needs a lot of configuration, not really a ready to play device.

**Comment by djfumberger** [comment]
> Now that the Retroid pocket 2 plus is coming out, maybe we should add some more systems to account for the new consoles the RP2+ can play (even if it can only play them poorly.)
> 
> * Sega Saturn
> * Nintendo Gamecube
> * Nintendo 3DS
>   Let me know if I'm forgetting any more consoles the RP2+ can play.

Yeah for sure, have pre-ordered the RP2+ and in the process of doing GameCube and Saturn.

**Comment by djfumberger** [comment]
> Well, new RP2+ will have its own frontend and launcher, so pegasus won't be necessary anymore.

Hopefully the new interface works well, but in any case always good to have options I think.

**Comment by sekuza009** [comment]
The interface is half way decent but still has a long way to go, nor does it look nearly as good as this theme for Pegasus. 

I'm wondering if you're willing to add Japanese versions of consoles though, like Super Famicom, Famicom, PC Engine, PC Engine SuperGrafx, Megadrive. As well as the expansions like the Sega 32x, Sega CD, TurboGrafx-CD. The way I have my roms is basically seperated into the different regions folders just so I can have my JP roms away from the USA sets.

There's also a few less common systems I'm not seeing that may be worth looking into adding, like the Pokemon Mini, MSX 2, MSX, Atari 5200, PC98.

The last question I had for you which I'm not sure is a thing with Pegasus or just the theme itself would be is there an ability to add folders into the game list? I like having sub folders for translations, hacks, all that. Thanks a ton and keep up the good work!

**Comment by ApolloFGC** [comment]
I want to add in Gamecube and 3DO myself, but I can't open a lot of these files to see how to do so. Hopefully a new update to the theme soon?
